backend/migrations/versions/af702716568f_add-share_health_columns.py

```diff
@@ -0,0 +1,30 @@
+"""_describe_changes_shortly
```

Comment:
Very minor, the alembic name of this migration is `describe_changes_shortly` which might be confusing when debugging. my guess is that you created the script with the instructions but the name in alembic stays :(